<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>750</width>
     <height>550</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>750</width>
     <height>550</height>
    </size>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>750</width>
      <height>550</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>750</width>
      <height>550</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>750</width>
      <height>550</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="start">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="send_btn">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="receive_btn">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Receive</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="send">
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="0" column="0">
       <widget class="QGroupBox" name="connectionGroupBox">
        <property name="minimumSize">
         <size>
          <width>700</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>700</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Connection</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0" colspan="3">
          <widget class="QLabel" name="connectionLbl">
           <property name="minimumSize">
            <size>
             <width>650</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>650</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>281</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="disconnectBtn">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Disconnect</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>281</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="receive">
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <widget class="QGroupBox" name="searchGroupBox">
        <property name="minimumSize">
         <size>
          <width>700</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>700</width>
          <height>150</height>
         </size>
        </property>
        <property name="title">
         <string>Search</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="4" column="0">
          <spacer name="verticalSpacer_14">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="searchInput_2">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>23</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="ipRadio_2">
           <property name="minimumSize">
            <size>
             <width>145</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>145</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Local IP Address</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="searchBtn_2">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="networkRadio_2">
           <property name="minimumSize">
            <size>
             <width>145</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>145</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Scan Network</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="nameRadio_2">
           <property name="minimumSize">
            <size>
             <width>145</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>145</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Computer Name</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QComboBox" name="optionsCombo_2">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>23</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QPushButton" name="connectBtn_2">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QRadioButton" name="previousRadio_2">
           <property name="minimumSize">
            <size>
             <width>145</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>145</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Previous Connections</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <spacer name="verticalSpacer_15">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="2">
          <spacer name="verticalSpacer_16">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="2">
          <spacer name="verticalSpacer_17">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="3">
          <spacer name="verticalSpacer_18">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="searchLbl_2">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Search:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="selectLbl_2">
           <property name="text">
            <string>Select:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="connectionGroupBox_2">
        <property name="minimumSize">
         <size>
          <width>700</width>
          <height>120</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>700</width>
          <height>120</height>
         </size>
        </property>
        <property name="title">
         <string>Connection</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0" colspan="3">
          <widget class="QLabel" name="connectionLbl_2">
           <property name="minimumSize">
            <size>
             <width>650</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>700</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>281</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="disconnectBtn_2">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Disconnect</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>281</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="syncGroupBox">
        <property name="minimumSize">
         <size>
          <width>700</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>700</width>
          <height>180</height>
         </size>
        </property>
        <property name="title">
         <string>Sync</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="5" column="6">
          <spacer name="verticalSpacer_10">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="folderLbl_2">
           <property name="text">
            <string>Music folder:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>118</width>
             <height>15</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="2">
          <widget class="QComboBox" name="drivesComboBox_2">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QPushButton" name="rbBtn_2">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Sync Rekordbox only</string>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <spacer name="verticalSpacer_11">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="2" colspan="3">
          <widget class="QLineEdit" name="folderInput_2">
           <property name="minimumSize">
            <size>
             <width>382</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>382</width>
             <height>23</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QPushButton" name="audioBtn_2">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Sync Audio Files only</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="drivesLbl_2">
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Rekorbox Data Drive:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="5">
          <widget class="QPushButton" name="allBtn_2">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Sync All</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <spacer name="horizontalSpacer_11">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="4">
          <spacer name="horizontalSpacer_12">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>118</width>
             <height>15</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <spacer name="verticalSpacer_12">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="0">
          <spacer name="verticalSpacer_13">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="5">
          <widget class="QToolButton" name="musicFolderBtn_2">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
